configuration/sitemaps.md

```diff
@@ -354,12 +360,12 @@ Image [item=<itemname>] [icon="<iconname>"] url="<url of image>" [label="<labeln
 ```
 
 This element type is able to present an image.
-The image has to be available on a reachable website or webserver without password or access token.
-It's also possible to place an image in the `html` folder under your configuration folder.
+The image must be available on a reachable website or webserver without password or access token.
+It's also possible to place an image in the `$OPENHAB_CONF/html` folder.
 The file will be available under the "static" route, [http://<my.openHAB.device>:8080/static/image.png](http://127.0.0.1:8080/static).
```

Comment:
"Such a file" ?

Reply:
Um... how about replacing from "It's also possible..." with,

"Alternatively, the image may be stored locally in the `$OPENHAB_CONF/html` folder, and will be accessible through the static route, [http://<my.openHAB.device>:8080/static/image.png](http://127.0.0.1:8080/static)."

Reply:
Sounds good.
In other articles we used  http://openhab-device:8080 and upon reading the sentence, it feels like image.png comes a bit out of nothing... not sure how to make clear that the name is an example.

Reply:
Here is what I will put in my next commit:

Alternatively, the image file (e.g. YourImageFile.png) may be stored locally in the $OPENHAB_CONF/html folder, and will be accessible through the static route, http://<my.openHAB.device>:8080/static/YourImageFile.png.

Reply:
This comment has been addressed.